.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "workbench.colorTheme": "Quiet Light"
```

Comment:
cool

Reply:
Whoops! Meant to take that out lol! But glad you like it!